File: 412-Fizz_Buzz/solution.cs
public class Solution {
    public IList<string> FizzBuzz(int n) {
        List<string> answer = new List<string>();

        for (int i = 1; i <= n; i++)
        {
            bool divisibleby3 = i % 3 == 0;
            bool divisibleby5 = i % 5 == 0;

            string currStr = "";
            if(divisibleby3)
            {
                currStr += "Fizz";
            }
            if(divisibleby5)
            {
                currStr += "Buzz";
            }
            if(currStr.Equals(""))
            {
                currStr = i.ToString();
            }
            answer.Add(currStr);
        }
        return answer;
    }//Time Complexity = O(n)
    //Space Complexity = O(1)
}

File: 412-Fizz_Buzz/solution.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        answer = []
        for i in range (1,n+1):
            divisibleby3 = i % 3 == 0
            divisibleby5 = i % 5 == 0

            currStr = ""
            if divisibleby3:
                currStr += "Fizz"
            if divisibleby5:
                currStr += "Buzz"
            if not currStr:
                currStr += str(i)

            answer.append(currStr)
        return answer